README.md

```diff
@@ -1,66 +1,108 @@
-1. Install docker toolbox from https://www.docker.com/products/docker-toolbox
-2. Run the docker quickstart terminal
-3. Change default vm settings (Only needed once)
-  1. Remove the default vm
+## Working with Network Proxies
```

Comment:
I'm not sure if I would put this at the very top because for a good portion of people it will just be noise. Usually the first section is just a "what is this" section.